packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -102,45 +112,54 @@ export default function CardSection({
               isSection
             />
           </button>
-          {actions && <Actions actions={actions} />}
+          {actions && (
+            <Stack inline grow={false} justify="end">
+              {actions}
+            </Stack>
+          )}
         </div>
       )}
 
       {(title != null || header != null) && !expandable && (
-        <div className="p-400 lm:p-600 w-full">
-          <Header
-            title={title}
-            titleAs={titleAs}
-            description={description}
-            expandable={expandable}
-            header={header}
-            expanded={opened}
-            actions={actions}
-            isSection
-          />
-          {actions && <Actions actions={actions} />}
+        <div className="p-400 lm:p-600 ">
+          <InteractiveWrapper onClick={!children ? onClick : undefined}>
```

Comment:
I added this `!children` condition, bcs in case that CardSection has both - header and children and `onClick`, then the CardSection component is focusable twice.